src/test/uint256_tests.cpp

```diff
@@ -58,13 +58,7 @@ static std::string ArrayToString(const unsigned char A[], unsigned int width)
     return Stream.str();
 }
 
-inline uint160 uint160S(const char *str)
-{
-    uint160 rv;
-    rv.SetHex(str);
-    return rv;
-}
-inline uint160 uint160S(const std::string& str)
+inline uint160 uint160S(std::string_view str)
```

Comment:
See https://github.com/bitcoin/bitcoin/pull/30436#discussion_r1686533487

Reply:
I'm not sure why the type is relevant here, what I'm saying is that we had a [const](https://learn.microsoft.com/en-us/cpp/cpp/const-cpp) guarantee before which we can keep.

Reply:
it's not very important in this particular method, but `inline` + `const` could help with the actual inlining - it's also clearer to the reader to understand what happens when mutating the parameter, since after inlining it should ideally avoid the copying (which is easy to do for a const).
At least this was my thinking for recommending it, let me know if I'm mistaken.